index.bs

```diff
@@ -502,6 +469,85 @@ ReadableStream(<var>underlyingSource</var> = {}, { <var>size</var>, <var>highWat
   1. Otherwise, throw a *RangeError* exception.
 </emu-alg>
 
+<h4 id="underlying-source-api">Underlying Source API</h4>
+
+The {{ReadableStream()}} constructor accepts as its first argument a JavaScript object representing the <a>underlying
+source</a>. Such objects may contain any of the following properties:
+
+<dl>
+  <dt><dfn method for="underlying source">start(<var>controller</var>)</dfn></dt>
```

Comment:
`controller` is not defined in this section. At the very least there should have a paragraph similar to the one in “Underlying Sink API”.

Reply:
It's defined in the section on `type`. I'll move it down to the bottom, to be symmetric with writable streams